pandas/tests/test_indexing.py

```diff
@@ -5619,19 +5650,12 @@ def f():
                           slice(3, 4.0),
                           slice(3.0, 4.0)]:
 
-                    def f():
-                        s.iloc[l]
-                    self.assertRaises(TypeError, f)
-
-                    def f():
-                        s.loc[l]
-                    self.assertRaises(TypeError, f)
+                    for idxr in [lambda x: x.loc,
+                                 lambda x: x.iloc,
+                                 lambda x: x]:
+                        self.assertRaises(TypeError, lambda: idxr(s)[l])
```

Comment:
these are about slicing:

In [1]: s = Series(range(3))

In [2]: s[1.0:2]
TypeError: cannot do slice start value indexing on <class 'pandas.indexes.range.RangeIndex'> with these indexers [1.0] of <type 'float'>

In [3]: s.loc[1.0:2]
TypeError: cannot do slice indexing on <class 'pandas.indexes.range.RangeIndex'> with these indexers [1.0] of <type 'float'>

In [4]: s.ix[1.0:2]
Out[4]: 
1    1
2    2
dtype: int64
```